secrets/admiral.yml

```diff
@@ -37,9 +37,38 @@ dev-mode:
     - admiral.port_scan
     - admiral.tester
 
-cert-worker: &default-section
+# The sslmate API offers 3 pricing tiers: Small, Medium, and Large. We make
+# Small the default because it's free. See https://sslmate.com/ct_search_api/
+# for more details.
+cert-worker-small: &default-section
+  celery:
+    <<: *celery-defaults
+    # 10 full-domain queries per hour
+    task_annotations: {'tasks.summary_by_domain': '10/h'}
+    task_default_queue: cyhy_cert_work
+    task_queues:
+      cyhy_cert_work:
+        routing_key: cyhy_cert_work
+  autodiscover_tasks:
+    - admiral.certs
+
+cert-worker-medium:
+  celery:
+    <<: *celery-defaults
+    # 100 full-domain queries per hour
+    task_annotations: {'tasks.summary_by_domain': '100/h'}
+    task_default_queue: cyhy_cert_work
+    task_queues:
+      cyhy_cert_work:
+        routing_key: cyhy_cert_work
+  autodiscover_tasks:
+    - admiral.certs
+
+cert-worker-large:
   celery:
     <<: *celery-defaults
+    # 1000 full-domain queries per hour
+    task_annotations: {'tasks.summary_by_domain': '1000/h'}
```

Comment:
Instead of making new `cert-worker-small`/`cert-worker-medium`/`cert-worker-large` sections, why not just make the rate limit itself the configurable item?

```suggestion
task_rate_limit: &task_rate_limit
  # Modify the rate below as needed.  "10/h" means limit to 10 requests per hour.
  summary_by_domain: 10/h

cert-worker:
  celery:
    <<: *celery-defaults
    task_annotations:
      tasks: *task_rate_limit
```

This allows you to remove the comments above that are specific to the various sslmate tiers and keep everything more generalized to the number of requests per hour.

Reply:
Looks great! 👍